src/components/Launch/LaunchWorkflowForm/test/LaunchWorkflowForm.test.tsx

```diff
@@ -287,7 +289,9 @@ describe('LaunchWorkflowForm', () => {
             expect(integerInput).toBeInvalid();
 
             fireEvent.change(integerInput, { target: { value: '123' } });
-            act(jest.runAllTimers);
+            act(() => {
+                jest.runAllTimers;
+            });
             await wait();
             expect(integerInput).toBeValid();
```

Comment:
This assertion seems to be failing in CI

Reply:
Along with the assertion on line 472 of this file

Reply:
Yep, looking at those failures now 👀